js/src/ui/Page/page.css

```diff
@@ -18,7 +18,7 @@
 .layout {
   padding: 0.25em 0.25em 1em 0.25em;
 
-  > * {
-    margin-bottom: 0.75em;
+  &>div {
```

Comment:
Swapped to margin.

I recall now. The reasson why this was originally introduced (not this PR actually, this was just supposed to move stuff around) it to not make the fixed statusbar at the bottom overlap with the page. Any other solution and you start having funnies across the different pages, e.g. dapps as well.

May be good to have a re-look at this in your fixes branch.